doc/changelog.rst

```diff
@@ -3,6 +3,12 @@ Changelog
 
 .. currentmodule:: motor.motor_tornado
 
+Motor 2.2
+---------
+
+Motor 2.2 adds support for MongoDB 4.4 features. It depends on PyMongo 3.11 or
+later. Motor continues to support MongoDB 3.0 and later.
```

Comment:
Mention that it drops support for Python 2.7 and Python 3.4?

Reply:
Done.